packages/cli/package.json

```
@@ -84,6 +84,7 @@
     "@types/which": "1.3.2",
     "@types/write-json-file": "2.2.1",
     "@types/yauzl-promise": "2.1.0",
+    "@vercel-internals/constants": "workspace:*",
```

Drop `workspace:` to be consistent with the others?

```suggestion
    "@vercel-internals/constants": "*",
```